spec/order_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'order'
+
+describe Order do
+
+  describe '#add_item' do
+    context '1 item in the order' do
+      it 'adds an item to be printed later' do
+        item_dbl = double("Burger", :Burger => "£4")
+        order = Order.new
```

Comment:
You could add any doubles you plan on using for a specific method between the describe and the context so you don't have to repeat the code in other tests (e.g. check the total_per_dish adds up when you add a dish a second time